model/src/main/java/org/cbioportal/model/ClinicalAttribute.java

```diff
@@ -0,0 +1,61 @@
+package org.cbioportal.model;
+
+import java.io.Serializable;
+
+public class ClinicalAttribute implements Serializable {
```

Comment:
it really paid of for us at ICGC but it is definitely not without headaches at time. Still, it has a few annotations that truly make developing in java less painful (notably the `@Value`, `val` and the `@XXXConstructor` ones)